datahub-classify/src/datahub_classify/similarity_predictor.py

```diff
@@ -0,0 +1,640 @@
+import logging
+import os
+import re
+from typing import Dict, List, Optional, Tuple
+
+import numpy as np
+
+import datahub_classify.similarity_numeric_constants as config
+from datahub_classify.helper_classes import (
+    ColumnInfo,
+    FactorDebugInfo,
+    SimilarityDebugInfo,
+    SimilarityInfo,
+    TableInfo,
+    TextEmbeddings,
+)
+from datahub_classify.utils import (
+    compute_string_similarity,
+    get_fuzzy_score,
+    load_stopwords,
+    read_glove_vector,
+)
+
+logger = logging.getLogger(__name__)
+current_wdr = os.path.dirname(os.path.abspath(__file__))
+glove_vec = os.path.join(current_wdr, "glove.6B.50d.txt")
+stop_words = load_stopwords()
+
+try:
+    from sentence_transformers import SentenceTransformer
+
+    model = SentenceTransformer("all-MiniLM-L6-v2")
+except Exception as e:
+    print(e)
```

Comment:
Change to logger.

Reply:
Modified the code to use logger instead of print